README.md

```diff
@@ -144,6 +144,7 @@ Options passed to the plugin constructor will overwrite options from the cosmico
 | ----------------- | --------------------- | ------------------------------------------------ | ----------- |
 | `async`           | `boolean`             | `compiler.options.mode === 'development'`        | If `true`, reports issues **after** webpack's compilation is done. Thanks to that it doesn't block the compilation. Used only in the `watch` mode. | 
 | `typescript`      | `object` or `boolean` | `true`                                           | If a `boolean`, it enables/disables TypeScript checker. If an `object`, see [TypeScript options](#typescript-options). |
+| `typescriptPath`  | `string`              | `require.resolve('typescript')`                  | If supplied this is a custom path where TypeScript can be found. |
```

Comment:
please move typescriptPath to the typescript options and use the same interface as other typescript options :)

Reply:
I moved it as requested, the integration seems cleaner now :)